### DISCUSSION TITLE
How to disable the MacOS menu entirely?

### DISCUSSION THREAD

**Discussion by martin-braun** [wezterm/wezterm#4980] [Q&A] [answered]
### What Operating System(s) are you running on?

macOS

### WezTerm version

wezterm 20240128-202157-1e552d76

### Ask your question!

The entire MacOS menu is unnecessary to me:

<img width="404" alt="Screenshot 2024-02-07 at 17 19 00" src="https://github.com/wez/wezterm/assets/24259245/fa7cb356-b8e6-4364-b159-c02c7b300c0f">

I'd love to deactivate it, so much so, that only "WezTerm" (and probably "Window" with the macOS tile left/right screen stuff, stuff that is native to macOS) is visible.

- I only use copy and paste macOS functions via hotkeys, my entire tab and pane work flow is `tmux`. I don't want any of this "fancy" stuff and I would even welcome a solution that disables all the features in that menu as well (except for copy/paste)
- The "Shell" menu is insanely polluted, because I have already 300 SSH aliases configured in my ssh config (across multiple files that are all included). I plan to get way more connections and the "Shell" menu is as risk of causing slow downs to WezTerm

Can I please get rid of all those menu entries (and their underlying functions). A bare bone terminal with copy and paste support is all that I seek for.

**Comment by wez** [comment, marked as answer]
You can either ignore this or switch to some other bare bones terminal emulator.

**Reply by martin-braun** [reply]
I get what you are trying to say.
